### DISCUSSION TITLE
New ideas

### DISCUSSION THREAD

**Discussion by Hard844** [MissingCore/Music#78] [Ideas]
First and foremost, I want to commend you on creating such a fantastic music application! The user interface is clean, and the overall experience is enjoyable. It's clear that a lot of thought and effort has gone into the development, and I’m excited to see how the app evolves.

I understand that the app is still in its early stages, but I wanted to suggest a few features that could enhance the user experience even further:

1. Swiping Gestures: It would be great if users could swipe left or right to change tracks or jump to the next/previous song. This would make navigation smoother and more intuitive.
  
2. Gradient Backgrounds:Adding an option for a gradient background could give the app a more vibrant and modern look. It could also adapt to the album art colors for a cohesive visual experience.
  
3. Multi-Select for Playlists:An option to select multiple songs and add them to a playlist all at once would be incredibly useful. This would streamline the process of creating and managing playlists.

I believe these features would add to the already excellent functionality of the app and further enhance user satisfaction. I appreciate your consideration of these suggestions and look forward to seeing how the app continues to develop.

**Comment by adilurrhmn** [comment]
Still offline lyrics not available

**Comment by cyanChill** [comment]
Improved playlist population will be added in the next major update. If you're using the APK version of the app, you can try it out early with the [`v2.0.0-rc.1`](https://github.com/MissingCore/Music/releases/tag/v2.0.0-rc.1) release.
- Instead of multi-select, we opted to adding a UI that allows you to search for tracks or albums that you want to add to a playlist.
- With swipe gestures, we did implement that on the home screen (so you can swipe between home/folders/playlists/tracks/albums/artists). On the current track screen, it'll probably be added in some point in the future.
- I currently don't have plans for gradient backgrounds as we want to keep the monochromatic look (though light mode is coming with this major update).
